file.go

```diff
@@ -117,6 +123,10 @@ func OpenFile(r io.ReaderAt, size int64, options ...FileOption) (*File, error) {
 						return nil, err
 					}
 					offset, _ = s.Seek(0, io.SeekCurrent)
+					if cast, ok := r.(interface{ SetBloomFilterSection(offset, length int64) }); ok {
```

Comment:
You are correct, maybe this would be the way to include the bloom filter header?
```go
bloomFilterOffset := c.chunk.MetaData.BloomFilterOffset
bloomFilterLength := (offset - bloomFilterOffset) + int64(h.NumBytes)
```

Reply:
🙌  that works perfectly. I know we've gone back and forth a few times - thanks for being patient as we worked through the metadata caching design!